sources/packages/backend/apps/api/src/route-controllers/application-exception/_tests_/e2e/application-exception.aest.getExceptionById.e2e-spec.ts

```diff
@@ -25,7 +25,8 @@ describe("ApplicationExceptionAESTController(e2e)-getExceptionById", () => {
     // Arrange
     const application = await saveFakeApplicationWithApplicationException(
       appDataSource,
-      ApplicationExceptionStatus.Approved,
+      undefined,
+      { applicationExceptionStatus: ApplicationExceptionStatus.Approved },
```

Comment:
Looking at the test case `Should get an application exception by id when available` what is the benefit of adding status here?

Reply:
this is not part of the PR TCs. it was already there, since I updated the parameter of the `saveFakeApplicationWithApplicationException` I edited this line/file

Reply:
Before `saveFakeApplicationWithApplicationException ` required an applicationExceptionStatus but with changes it's ok not pass it.

Reply:
Yes, we can remove this changes from part of the PR now.